src/pages/DelegatoryValidator/MyDepositsSection.tsx

```diff
@@ -178,14 +154,14 @@ function ActionsCell({handleClickOpen, status}: MyDepositsSectionCellProps) {
     }
   }
   return (
-    <GeneralTableCell sx={{textAlign: "right"}}>
+    <GeneralTableCell sx={{textAlign: "right", paddingRight: 3}}>
       <Button
         variant="primary"
         size="small"
         onClick={handleClickOpen}
-        sx={{maxWidth: "10%", paddingY: 1}}
+        sx={{width: "30px", maxHeight: "40px"}}
```

Comment:
what's the difference between 10% and 30px. I imagine one being relative, does that mean it is responsive better when we have different screen size?

Reply:
with limited frontend knowledge as well haha, percentage should respond better with different screen change, but there for the button width, we want something more fixed cuz all other components are relatively fixed in size as well when screen size is bigger than `!useMediaQuery(theme.breakpoints.up("md"))`.